R/flatten.R

```diff
@@ -31,7 +31,7 @@
 #' Then all involved foreign tables are joined to the `start` table successively, with the join function given in the `join` argument.
 #'
 #' **Case 2**, filter conditions are set for at least one table that is connected to `start`:
-#' First, disambiguation will be performed if necessary. The `start` table is then calculated using `tbl(dm, "start")`.
+#' First, disambiguation will be performed if necessary. The `start` table is then calculated using `dm[["start"]]`.
```

Comment:
`start` is an argument to this function:?

```suggestion
#' First, disambiguation will be performed if necessary. The `start` table is then calculated using `dm[[start]]`.
```